### DISCUSSION TITLE
.net core

### DISCUSSION THREAD

**Discussion by sajjadarashhh** [dnnsoftware/Dnn.Platform#5261] [Q&A]
When we have .net core Version of DNN?

**Comment by bdukes** [comment]
We do not currently intend to have a version of DNN which supports .NET Core (or .NET 5+). You can see https://dnncommunity.org/blogs/Post/7625/The-Technical-Future-of-DNN and https://dnncommunity.org/blogs/Post/7626/Strategy-To-Know-Thyself for details around that decision. Hope it helps!

**Comment by sajjadarashhh** [comment]
@bdukes but we can to take new repo with .net core support and developing parallel thats repos and see which of them are better!
I've read thats posts in past months and read something problems like .net core lifecycle and etc...
thats true but this not bad!
It is both good and bad It depends on how we look at it
if we just look at something like we need to change .net version of repo in every year this may be difficult it's bad!
but we look at that features in new versions of .net somethings like blazor, blazor hybrid and... 
and use them to create this features:
create applications from website with blazored hybrid,
create simple webUis without js and create simple editor for webUis,
nice performance,
and...
this can be very nice!
but I dont know why DNN Community avoiding to migrate .net core versions!

**Comment by valadas** [comment]
This project is open source and MIT licensed, anyone can fork it and do a .Net Core version of it. But:
1. It is massive work and although some efforts have been made in some areas to bring it to .Net Standard, it would take a huge amount of effort to do that everywhere.
2. DNN is very extensible and has many 3rd party solutions, most of those solutions would break instantly, killing the existing ecosystem.
